### ISSUE TITLE
Enter initial settings

### ISSUE THREAD

**Issue by wroughtec** [nathanwoulfe/PieMan#1]
![screen shot 2015-07-22 at 09 49 42](https://cloud.githubusercontent.com/assets/74549/8821623/1c62ae82-3057-11e5-83af-0c650a8bc203.png)
I added a datatype however I can not see how I can enter my analytics profile?
Also getting:
Request error: The URL returned a 404 (not found): 
backoffice/pieman/editorapi/getprevalues

on the pages with this dataype

Any suggestions
Umbraco 7.2.8 
PieMan version 0.2

**Comment by nathanwoulfe** [comment]
Looks like you haven't authorised the package - once you've added it to a document type, open a node of that type in the content tree. 

The PieMan editor will have a gear icon, click that to open the settings. 

At the top of the settings is an authorize button. Click that one and complete the oAuth flow. 

Refresh the back office, open the settings again and you should have options in the two drop downs. Select the account and profile, save the changes and publish the page. 

That should do it. It's a little convoluted, but is a one-off process. 

The 404, off the top of my head, is probably related to not having done the oAuth steps. 

Let me know how you get on. 

N

**Comment by wroughtec** [comment]
![screen shot 2015-07-22 at 10 24 14](https://cloud.githubusercontent.com/assets/74549/8822239/d9a3a3c6-305b-11e5-8b65-fd8a80deb8c0.png)

hmm I am not seeing a authorize button

**Comment by nathanwoulfe** [comment]
That's a bit strange. Looks like there's a JavaScript issue given the {{}} braces. 

I haven't tested on 7.2.8, so it's possible there's something version-related. I'll spin up a site and have a look. 

Have you tried restarting the site / bumping web.config?

**Comment by wroughtec** [comment]
yes tried that here is my console errors:

```
Failed to load resource: the server responded with a status of 404 (Not Found)
5umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:63 Error: $digest already in progress
    at Error (native)
    at g (http://localhost:64091/umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:87:150)
    at Object.e.$apply (http://localhost:64091/umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:92:362)
    at http://localhost:64091/App_Plugins/Epiphany.SeoMetadata/js/epiphany.seo.metadata.js?cdv=802199329:98:14
    at x.event.dispatch (http://localhost:64091/umbraco/lib/jquery/jquery.min.js?cdv=802199329:5:10006)
    at y.handle (http://localhost:64091/umbraco/lib/jquery/jquery.min.js?cdv=802199329:5:6789)
    at Object.x.event.trigger (http://localhost:64091/umbraco/lib/jquery/jquery.min.js?cdv=802199329:5:9131)
    at http://localhost:64091/umbraco/lib/jquery/jquery.min.js?cdv=802199329:5:14836
    at Function.x.extend.each (http://localhost:64091/umbraco/lib/jquery/jquery.min.js?cdv=802199329:4:4575)
    at x.fn.x.each (http://localhost:64091/umbraco/lib/jquery/jquery.min.js?cdv=802199329:4:1626)(anonymous function) @ umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:63
http://localhost:64091/umbraco/backoffice/pieman/editorapi/getprevalues Failed to load resource: the server responded with a status of 404 (Not Found)
umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:63 TypeError: Cannot read property 'refresh_token' of undefined
    at new <anonymous> (App_Plugins/PieMan/controllers/PieMan.settingsController.js?cdv=802199329:8)
    at d (umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:28)
    at Object.instantiate (umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:28)
    at umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:53
    at umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:44
    at n (umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:7)
    at k (umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:44)
    at e (umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:40)
    at k (umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:44)
    at e (umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:40)(anonymous function) @ umbraco/lib/angular/1.1.5/angular.min.js?cdv=802199329:63
```

**Comment by nathanwoulfe** [comment]
Looks like the DLL is missing from the package - which is pretty strange given you're the first to raise the issue despite plenty of downloads. I've just tried downloading both versions from the Umbraco repo and they are incomplete. Will need to look at it at work tomorrow morning.

**Comment by nathanwoulfe** [comment]
Hi William

Looks like I had a bit of a stuff-up and forgot to include PieMan.dll in the v0.2.0 package. Goes a long way to explaining why it wasn't working. I've just released a [v0.3.0 package](https://our.umbraco.org/projects/backoffice-extensions/pieman-analytics/), please uninstall v0.2.0 and reinstall with the lastest version. I've tested on Umbraco 7.2.4 with no issues. 

I'll also add the compiled dll to the repo so you can just download that and add it to your site.

N

**Comment by wroughtec** [comment]
Seems to be working now :+1:
